Source/Core/DolphinQt/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -589,6 +611,10 @@ void EnhancementsWidget::AddDescriptions()
   static const char TR_3D_SWAP_EYES_DESCRIPTION[] = QT_TR_NOOP(
       "Swaps the left and right eye. Most useful in side-by-side stereoscopy "
       "mode.<br><br><dolphin_emphasis>If unsure, leave this unchecked.</dolphin_emphasis>");
+  static const char TR_3D_PER_EYE_RESOLUTION_DESCRIPTION[] =
+      QT_TR_NOOP("Whether each eye gets full or half image resolution when using side-by-side "
+                 "or above-and-below 3D."
+                 "<br><br><dolphin_emphasis>If unsure, leave set to full.</dolphin_emphasis>");
```

Comment:
Just for consistency with other descriptions:

```suggestion
                 "<br><br><dolphin_emphasis>If unsure, select Full.</dolphin_emphasis>");
```

Reply:
Also, given that Half has been the default for years, the "if unsure" recommendation should probably be Half.

Reply:
> Also, given that Half has been the default for years, the "if unsure" recommendation should probably be Half.

I disagree. I'm willing to change this if you feel strongly about it, but...

Today, if you buy a device that supports side-by-side 3D, it likely does so with two identical screens. So far as I know there aren't more "half SBS" products being produced. If you select "Half" with one of those devices, what you get is something that _sort of_ looks right - it's 3D... but you're not only losing half the resolution in the visible content area, you've also got letterboxing meaning the content doesn't occupy your viewport properly.

I think we should recommend to those who are unsure that they select Full, and if someone knows they have a half-SBS device (and how could they not know?!), they'll know well enough to ignore the default.

Reply:
I'm indifferent to the default.  My thought is that people that partake in stereo 3d are usually quite tech savy.  If they see the wrong 3d output, they will likely understand they need to mess with the settings.  Additionally, given most of the new tech devices that offer 3d (Lume Pad, Looking Glass, some of the special monitors with eye tracking) don't work with Dolphin, most of the tech being used is going to be older regardless of which default we choose.

Reply:
I will say, that this should probably match the default.  Both the default of the enum and the default in the settings is 'Half'.  I didn't do a full look but it would seem odd to have a different default than what is recommended.

Reply:
This was exactly what I was trying to say. The current Dolphin behaviour is "Half". But I think it's better to use Full, because using Full is more commonly the correct setting (and as time goes on, Half will be the right setting less and less often).

I left the config defaulted to Half to preserve the existing behaviour, but recommended the use of Full for that reason.

I don't particularly mind if we change either the default setting (breaking backwards compat) or the recommendation (recommending users do something that will be less likely the right thing as the years go by). Whatever you like.